changelog

```diff
@@ -1,6 +1,7 @@
 MSAL Wiki : https://github.com/AzureAD/microsoft-authentication-library-for-android/wiki
 V.Next
 ----------
+- [MAJOR] Support CIAM Authority Type (#1783)
```

Comment:
Why should this be a MAJOR?

Reply:
Doesn't have to be, seems like a pretty big offering to me, as we are opening up to a new customer base, that's why i figured it would be Major.

Reply:
I'll go with Minor since it's not a breaking change. CIAM is not in Build yet anyway, so this change is not actually enabling any new customers yet, not until CIAM tenants are fully available.